### ISSUE TITLE
2.4.7 Visible Focus

### ISSUE THREAD

**Issue by renujain31** [ushahidi/platform#4185]
When an interactive element (link, button, form field, selectable element, etc.) receives focus, a visual indicator shows so a user can see what element they are currently on.
We should provide a strong, contrasting colour for a box border when it is in focus.

Related issue: #4149

**Comment by lexyd** [comment]
Hi @tuxpiper I have designed a solution for this issue raised by @renujain31. 

I designed an OnHover effect button.
All designs and colour are in line with the existing pattern library.

Implementation should be easy and seamless. I have attached an image of the solution

<img width="946" alt="Ushahidi Button solution" src="https://user-images.githubusercontent.com/17413130/185432674-6bc53cac-c144-4e2e-88ed-7322ceb21e8a.png">
